### ISSUE TITLE
Delete users

### ISSUE THREAD

**Issue by wintermeyer** [wort-schule/wort.schule#44]
An admin should be able to delete a user. We need to add a feature for this.

@phylor Does this interact with the history? Can we delete a user who is in the history of a word without raising some sort of error for that word?

**Comment by phylor** [comment]
> @phylor Does this interact with the history?

No, not really. Papertrail stores the ID of the user as a string (without a foreign key). That means that you have to take care when reading the value, as the ID might not be valid anymore.

> Can we delete a user who is in the history of a word without raising some sort of error for that word?

In terms of history, yes. We do have to think about other objects though: for example, if we delete a teacher, what happens to learning groups of that teacher? It's probably not that hard, but we should think through all the possible objects and define a behavior for each of them.

**Comment by wintermeyer** [comment]
I did not think about the teachers. 🤦‍♂️

I hibernate this issue for now. We have to tackle it some day but not today.
